pkg/operator/operator.go

```diff
@@ -175,21 +174,18 @@ func (o *Operator) Start(ctx context.Context) {
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
-			sharedmain.MainWithConfig(ctx, "webhook", o.GetConfig(), o.webhooks...)
+			sharedmain.MainWithConfig(sharedmain.WithHealthProbesDisabled(ctx), "webhook", o.GetConfig(), o.webhooks...)
 		}()
 	}
 	wg.Wait()
 }
 
-func knativeChecker(path string) healthz.Checker {
+func webhookChecker(ctx context.Context) healthz.Checker {
 	return func(req *http.Request) (err error) {
-		res, err := http.Get(fmt.Sprintf("http://localhost:%d/%s", knativeinjection.HealthCheckDefaultPort, path))
+		_, err = http.Get(fmt.Sprintf("http://localhost:%d", injection.GetOptions(ctx).WebhookPort))
 		if err != nil {
 			return err
 		}
-		if res.StatusCode != http.StatusOK {
```

Comment:
nit: Can you add a TODO to add this back in when we're able to configure the health check ports?